versioneer.py

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
See https://github.com/dask/distributed/pull/4741#discussion_r660567523 a short discussion about the versioneer file.

I don't have a strong opinion

Reply:
+1 I would prefer to not touch our versioneer-related files. Does `pyupgrade` have an option to exclude a specified set of files?

Reply:
It exposes a whitelist of files, but not a blacklist. So I could exclude distributed/versioneer.py but I can't exclude distributed/distributed/_version.py.